### PR TITLE
Search and edit

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by robmoffat** [comment]
You can check the search in the preview, however until this PR is merged there won't be any search results as we need the sitemap working too

**Comment by robmoffat** [comment]
@psmulovics can you approve this one?  Can't really test the search bar out without fixing the sitemap too sadly

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -24,6 +23,7 @@ module.exports={
           "showLastUpdateTime": false,
           "path": "../docs",
           "sidebarPath": "sidebars.json",
+          "editUrl": "https://github.com/finos/open-source-readiness/edit/main/docs"
```
Comment (psmulovics) [inline website/docusaurus.config.js:26]: This is a nice addition!

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -36,6 +36,11 @@ module.exports={
     './src/plugin/category-listing'
   ],
   "themeConfig": {
+    "algolia": {
+      "appId": "LOYCC7X3U6",
```
Comment (psmulovics) [inline website/docusaurus.config.js:40]: Don't we want to keep these in SECRET?

Comment (robmoffat) [reply website/docusaurus.config.js:40]: put apiKey into a variable

**Pull request merged**